manim/mobject/types/point_cloud_mobject.py

```diff
@@ -271,6 +271,26 @@ def generate_points(self):
 
 
 class Point(PMobject):
+    """A point.
```

Comment:
I don't think is helpful. Can you remove this or add some more lines explaining what this does?

Reply:
Removed this